components/nexus-component-source/src/main/java/org/sonatype/nexus/component/source/config/ComponentSourceRegistryInitializedEvent.java

```diff
@@ -21,11 +20,8 @@
  * @since 3.0
  */
 public class ComponentSourceRegistryInitializedEvent
-    extends AbstractEvent<ComponentSourceRegistry>
 {
-  public ComponentSourceRegistryInitializedEvent(
-      final ComponentSourceRegistry component)
+  public ComponentSourceRegistryInitializedEvent(final ComponentSourceRegistry component)
   {
-    super(component);
```

Comment:
I didn't want to have to go change everywhere an event was created just to tidy up the underlying framework, which is why I left the ctors asis.